src/seeds/users.seed.ts

```diff
@@ -0,0 +1,21 @@
+import { randEmail, randFullName, randPassword, randUuid } from '@ngneat/falso'
+import { AppDataSource } from '../data-source'
+import { User } from '../entities/User'
+import { hash } from 'bcryptjs'
+
+export async function seedUsers() {
+  const userRepository = AppDataSource.getRepository(User)
+  const usersToInsert: User[] = []
+
+  for (let i = 0; i < 50; i++) {
+    usersToInsert.push({
+      id: randUuid(),
+      name: randFullName(),
+      email: randEmail(),
+      passwordHash: await hash(randPassword(), 6),
+      birthDate: '12-12-1990',
+    })
+  }
```

Comment:
como poderia fazer isso usando funções de array ao invés de um `for loop`

Reply:
acho que seria legal também investigar o porquê de darmos preferência a essas funções de array 🤔